package.json

```diff
@@ -59,6 +59,7 @@
     "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
     "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
     "jest": "jest",
+    "jest:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
Can `node_modules/.bin/jest` just be `jest`?

Reply:
I couldn’t get that to work.

Reply:
I was able to run with `node --inspect-brk jest --runInBand`. We can keep it this way.